php/Makefile

```diff
@@ -39,12 +29,13 @@ clean: ## Remove all build artifacts and files generated by the acceptance tests
 
 .DELETE_ON_ERROR:
 
-acceptance: .built $(TOKENS) $(ASTS) $(PICKLES) $(ERRORS) $(SOURCES) ## Build acceptance test dir and compare results with reference
+acceptance: .built  ## Test parser against test data
```

Comment:
Not sure if this should be `.PHONY` now?

Reply:
You can remove the goal entirety.

Do update the `php/README.md` with info about the structure.